src/fuse_pointwise.cpp

```diff
@@ -13,6 +13,8 @@ inline namespace MIGRAPHX_INLINE_NS {
 
 static literal get_scalar(instruction_ref ins)
 {
+    if(ins->name() == "contiguous")
+        return get_scalar(ins->inputs().front());
```

Comment:
For cases like this(such as a scalar or broadcasted shape) we skip const-folding of contiguous as it makes the tensor use more memory(and is inefficient) after lowering. Whereas if we keep the contiguous there, it can be removed if the lowered operator allows broadcasted or scalar shapes as inputs, and the original tensor shape can be preserved.

Reply:
Also if folded the contiguous then we wouldn't know thats its scalar, which we try to embed scalars into the code generation.